examples/01_plotting/plot_surf_atlas.py

```diff
@@ -0,0 +1,58 @@
+"""
+Demonstrate basic loading and plotting of a cortical surface atlas.
+===================================================================
+The Destrieux parcellation (Destrieux et al, 2010) in fsaverage5 space as
+distributed with Freesurfer (Dale et al, 1999, Fischl et al, 1999) is used as
+the chosen atlas.
+
+The :func:`nilearn.plotting.plot_surf_roi` function is used
+to plot the parcellation on the pial surface.
+
+See :ref:`plotting` for more details.
+
+References
```

Comment:
For me citing Destrieux paper would be enough I guess. Since other papers are already included in description in ```destrieux_surface.rst```